lib/absinthe/schema.ex

```diff
@@ -544,6 +553,20 @@ defmodule Absinthe.Schema do
     |> Enum.map(&lookup_directive(schema, &1))
   end
 
+  def to_sdl(schema) do
```

Comment:
This is a nice function to have!

We should use it in `Mix.Tasks.Absinthe.Schema.Sdl` as well

https://github.com/absinthe-graphql/absinthe/blob/master/lib/mix/tasks/absinthe.schema.sdl.ex#L76-L82